docker-compose.yml

```diff
@@ -1,19 +1,21 @@
-version: '3.9'
+
+version: '3.3'
 
 services:
   postgres:
     image: postgres
     restart: always
     env_file: .postgres.env
+    ports:
+      - "5432:5432"
     volumes:
-      - 'postgres-data:/var/lib/postgres/data'
+      - '/srv/postgres/postgres-data:/var/lib/postgres/data'
```

Comment:
did you discuss it with nico ? this looks strange

Reply:
Yes i did

Reply:
Works for me, because it's just the example docker-compose.yml which is not required for the continuous deployment at all.